dango/lending/Cargo.toml

```diff
@@ -21,6 +21,7 @@ library = []
 anyhow                = { workspace = true }
 dango-account-factory = { workspace = true, features = ["library"] }
 dango-auth            = { workspace = true }
+dango-oracle          = { workspace = true, features = ["library"] }
```

Comment:
This dependency seems unnecessary

Reply:
Thanks. Perhaps worth adding cargo-machete to the just file.